crates/datafusion_ext/src/metrics.rs

```diff
@@ -161,22 +174,22 @@ pub trait DataSourceMetricsOptsType: 'static + Debug + Clone + Sync + Send {
 #[derive(Debug, Clone)]
 pub struct ReadWriteDataSourceMetricsOptsType;
 impl DataSourceMetricsOptsType for ReadWriteDataSourceMetricsOptsType {
-    const OPTS: DataSourceMetricsOpts = DataSourceMetricsOpts::read_write();
     const DISPLAY_NAME_PREFIX: &'static str = "ReadWrite";
+    const OPTS: DataSourceMetricsOpts = DataSourceMetricsOpts::read_write();
```

Comment:
If this is being sorted alphabetically by some rule, I'd prefer if we didn't since there's definitely cases where these are ordered according to what the developer thinks is important.